hypothesis-extra/hypothesis-pytest/src/hypothesispytest/__init__.py

```diff
@@ -19,17 +19,20 @@
 
 class StoringReporter(object):
 
-    def __init__(self):
+    def __init__(self, config):
+        self.config = config
         self.results = []
 
     def __call__(self, msg):
+        if self.config.getoption('capture', 'fd') == 'no':
+            print(msg)
```

Comment:
Perhaps I should be using the pytest_runtest_call hook instead. It looks like it's closer to what I thought was happening here. I confess to not being very good at pytest internals yet. :-)